pkg/interceptors/injector/interceptor_test.go

```diff
@@ -0,0 +1,66 @@
+package injector
+
+import (
+	v1apps "k8s.io/api/apps/v1"
+	"k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
`v12` is imho a quite confusing name for this package. Would you mind `s/v12/v1meta/g`?

Reply:
I would generally recommend to add the version for anything in `k8s.io/api/` and skip it for anything in `k8s.io/apimachinery/`.